src/test/e2e/api/admin/projects.e2e.test.ts

```diff
@@ -0,0 +1,31 @@
+import dbInit, { ITestDb } from '../../helpers/database-init';
+import { IUnleashTest, setupApp } from '../../helpers/test-helper';
+import getLogger from '../../../fixtures/no-logger';
+let db: ITestDb;
+let app: IUnleashTest;
+
+beforeAll(async () => {
```

Comment:
Any reason why not using the existing `projects.e2e.test.ts`?

Reply:
No, I did not see it, sense it was in a separate folder. I will fix.